clang/include/clang/Basic/CodeGenOptions.def

```diff
@@ -515,6 +430,8 @@ ENUM_CODEGENOPT(ZeroCallUsedRegs, llvm::ZeroCallUsedRegs::ZeroCallUsedRegsKind,
 /// non-deleting destructors. (No effect on Microsoft ABI.)
 CODEGENOPT(CtorDtorReturnThis, 1, 0)
 
+#include "DebugOptions.def"
```

Comment:
Do you think it would make sense to eventually treat these as separate top-level `.def` files?

Reply:
That was my initial idea, but it ballooned into a lot of code changes. Turns out the debug options are used in a lot of places. It could be done, but it would be a lot of churn.

Reply:
Got it. In that case I suggest leaving a FIXME behind.